tests/ExcitedStates_test.py

```diff
@@ -20,20 +20,21 @@
 ## along with adcc. If not, see <http://www.gnu.org/licenses/>.
 ##
 ## ---------------------------------------------------------------------
-import unittest
 import pytest
 from numpy.testing import assert_allclose
 
-from adcc.testdata.cache import cache
 from adcc.OneParticleOperator import OneParticleOperator
 from adcc.ExcitedStates import EnergyCorrection
-from .test_state_densities import Runners
 
+from .testdata_cache import testdata_cache
+
+
+class TestExcitedStates:
+    def test_excitation_view(self):
+        state = testdata_cache.adcc_states(
+            system="h2o_sto3g", method="adc2", case="gen", kind="singlet"
```

Comment:
Shouldn't CVS be tested here as well? And what about the other ADC methods? Before, everything was tested here.

Reply:
Here we don't actually test the implemented equations. We only test that the `Excitation` class gives the same result as the `ExcitedStates` class. Since `Excitation` just calls out to the `ExcitedStates` implementation, this test should in my opinion be independent of reference case, state kind and system (restricted/unrestricted). Therefore, I decided to only verify a single test case.